landlab/components/erosion_deposition/generalized_erosion_deposition.py

```diff
@@ -126,11 +134,9 @@ def __init__(
         self.initialize_output_fields()
 
         self._qs = grid.at_node["sediment__flux"]
+        self._qs_in = grid.at_node["sediment__influx"]
```

Comment:
`self._qs_in` is unused. I recommend either exposing it as a property (with a descriptive name) or removing it.

Reply:
`self._qs_in` is defined here because it is used by derived classes.

Reply:
In that case, how about making it public? That way, derived classes don't have to depend on the existence of a private variable. I think the safest way to do that would be through a property,
```python
@property
def sedflux_influx(self):
    return  self.grid.at_node["sediment__influx"]
```
Another option would be to drop the leading underscore and change the name to *qs_in* but then someone could accidentally change the pointer to the data.

Reply:
Agreed - changed as suggested